src/components/CheckEmailStatusPage/CheckEmailStatusPage.js

```diff
@@ -7,6 +7,7 @@ import Headline from '@folio/stripes-components/lib/Headline';
 
 import OrganizationLogo from '../OrganizationLogo/OrganizationLogo';
 import validateEmail from '../../validators/validateEmail/validateEmail';
```

Comment:
Uppercase or lower case in imports? Please be consistent

Reply:
I think this imports are fine. Because we are using CamelCase for class based components and lowerCamelCase for functions such as helpers or validators. Guys what do you think about this?

Reply:
Same point as Sasha's one. `validateEmail` is a validation function and the `OrganizationLogo` is a functional component that is why their namings are different

Reply:
Thanks